src/pages/settings/Security/TwoFactorAuth/VerifyPage.js

```diff
@@ -123,10 +125,13 @@ function VerifyPage(props) {
                         <Text style={styles.mt5}>{props.translate('twoFactorAuth.addKey')}</Text>
                         <View style={[styles.mt11, styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween]}>
                             {Boolean(props.account.twoFactorAuthSecretKey) && <Text>{splitSecretInChunks(props.account.twoFactorAuthSecretKey)}</Text>}
-                            <Button
+                            <PressableWithDelayToggle
                                 text={props.translate('twoFactorAuth.copy')}
+                                icon={Expensicons.Copy}
+                                inline={false}
                                 onPress={() => Clipboard.setString(props.account.twoFactorAuthSecretKey)}
-                                medium
+                                styles={[styles.button, styles.buttonMedium, styles.twoFactorAuthCodesButton]}
```

Comment:
Thanks for the change!
I wonder if we can remove the minimum width property from `styles.twoFactorAuthCodesButton` [here](https://github.com/Expensify/App/blob/f08d2fb6f1ee07bfb83fbae25df37bc5d65365f7/src/styles/styles.js#L2195) as button looks wider. @shawnborton would like to hear your thoughts

<img width="966" alt="Screenshot 2023-06-05 at 5 18 47 PM" src="https://github.com/Expensify/App/assets/32012005/d09c2f16-5001-46d8-bc3f-8560e9156392">

Reply:
@MonilBhavsar @BeeMargarida I believe this minWidth was there to keep the width of these 2 buttons consistent (per design). Maybe we could remove only from the copy button, not from the whole style definition

![Screenshot 2023-06-05 at 15 58 57](https://github.com/Expensify/App/assets/26878038/cd58029f-b40d-4df0-91a0-7b84fcac5185)

Reply:
Oops, just saw that Ana did exactly that. Ignore me!